deploy/docker/clickhouse-setup/docker-compose.yaml

```diff
@@ -1,6 +1,7 @@
 version: "2.4"
 
-x-clickhouse-defaults: &clickhouse-defaults
+x-clickhouse-defaults:
```

Comment:
is this added by VSCode formatter with custom changes from your editor config?

If so, this could likely be reverted by formatter by others.

Reply:
It's the default VScode formatter.